boringssl/Dockerfile

```diff
@@ -18,6 +18,5 @@ FROM ossfuzz/base-libfuzzer
 MAINTAINER mike.aizatsky@gmail.com
 RUN apt-get install -y cmake ninja-build golang
 
-ENV GIT_URL "https://boringssl.googlesource.com/boringssl"
-
+RUN git clone https://boringssl.googlesource.com/boringssl
```

Comment:
Any reason why you decided to go with doing the checkout in the Dockerfile, as opposed to specifying a CHECKOUT_COMMAND and keeping checkout_and_compile? It seems a bit weird to me that the image will get a fixed checkout of the repository.